jaworski_cellseg/__init__.py

```diff
@@ -1 +1,3 @@
-from .jwski_widget import create_combined_widget
\ No newline at end of file
+#from .jwski_widget import create_combined_widget
+"""jaworski-cellseg3 - napari plugin for jaworski-lab."""
+__version__ = "0.1.0"
```

Comment:
This should be in pyproject.toml

Reply:
Only in there or in addition? I think it's nice to be able to retrieve the version number programmatically.

Reply:
I thought the goal was only to specify a version here. If we want to be able to programmatically retrieve the version number, then we can do this here `__version__ = importlib.metadata.version(__package__ or __name__)` so we only have one source of truth

Reply:
@digicosmos86 you want the version only in the `pyproject.toml `or in both files?

Reply:
I'm guessing Paul is suggesting retrieving the version number from pyproject.toml (as it is already there) using that command. That way a change in version number would propagate over here.
```suggestion
from importlib.metadata import version
__version__ = version(__package__ or __name__)
```